user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Automatic_incident_tracking.md

```diff
@@ -182,11 +182,10 @@ In addition, the following configuration is needed in the *configuration.xml* fi
   </item>
   ```
 
-After editing and saving the configuration file, **restart the SLAnalytics process on the leader agent** to make sure the desired changes take effect.
```

Comment:
@finestracheride How do users know which is the leader Agent? Strictly speaking, all DMAs in a cluster are equals and there is no leader.

Reply:
The concept of Leader DataMiner ID is described in the above section "Automatic incident tracking configuration in System Center"